backend/pkg/plugins/plugins_test.go

```diff
@@ -370,10 +400,29 @@ func TestHandlePluginReload(t *testing.T) {
 	err := ch.Set(context.Background(), plugins.PluginRefreshKey, true)
 	require.NoError(t, err)
 
+	// err := ch.Set(context.Background(), plugins.PluginRefreshKey, false)
+	// require.NoError(t, err)
```

Comment:
Do we want to keep this portion commented out?

Reply:
Good catch.